### PR TITLE
Forwarder rtd

### PR DESCRIPTION
**Pull request by schandrika**
Updated forward historian documentation
Deleted unused init/config parameter (destination_instance_name) from forward historian

### CODE REVIEW

File: docs/source/agent-framework/historian-agents/forwarder/forward-historian.rst [hunk 1]
```diff
@@ -4,58 +4,273 @@
...
 .. code-block:: json
 
     {
-        "agentid": "forwarder",
-        "destination-vip": "tcp://127.0.0.1:22916",
-        "destination-serverkey": "<SOME_KEY>"
+        "destination-address": "https://centvolttron2:8443"
     }
 
+An example configuration with above parameters is available at  `services/core/ForwardHistorian/config_web_address`.
```
Comment (shwethanidd) [inline docs/source/agent-framework/historian-agents/forwarder/forward-historian.rst:96]: I don't see this file.

File: docs/source/agent-framework/historian-agents/forwarder/forward-historian.rst [hunk 2]
```diff
@@ -4,58 +4,273 @@
...
 
-Adding the configuration option below will limit the backup cache to `n` gigabytes.  This will help keep a hard drive
-from filling up if the agent is disconnected from its target for a long time.
+Optional Configurations
+-----------------------
 
-::
+The most common use case for a forwarder is to forward data collected on a local instance to a remote historian. Due to
```
Comment (shwethanidd) [inline docs/source/agent-framework/historian-agents/forwarder/forward-historian.rst:101]: Is there any other usecase?

Comment (schandrika) [reply docs/source/agent-framework/historian-agents/forwarder/forward-historian.rst:101]: In theory I can think of other cases where the target is not a historian -- Forward analysis result to a different machine for post processing with a software that is not on the collector box? Forward simulation result to another machine that is connected to a device that can be sent a set_point instruction based on simulation result ?

File: docs/source/agent-framework/historian-agents/forwarder/forward-historian.rst (hunk 1 above)
Comment (schandrika) [reply docs/source/agent-framework/historian-agents/forwarder/forward-historian.rst:96]: Added the example config file

File: docs/source/agent-framework/historian-agents/forwarder/forward-historian.rst (hunk 2 above)
Comment (shwethanidd) [reply docs/source/agent-framework/historian-agents/forwarder/forward-historian.rst:101]: Analysis/simulation result is also data collected right? I didn't quite understand by - "use case for a forwarder is to forward data collected on a local instance". Anyway will pull it in.

**Pull request merged**